src/main/java/org/openjdk/jextract/impl/HeaderFileBuilder.java

```diff
@@ -66,10 +66,9 @@ public void addVar(Declaration.Variable varTree, Optional<String> fiName) {
             String segmentConstant = emitGlobalSegment(layoutVar, javaName, nativeName, null);
             emitGlobalGetter(segmentConstant, vhConstant, javaName, varTree, "Getter for variable:");
             emitGlobalSetter(segmentConstant, vhConstant, javaName, varTree, "Setter for variable:");
-
-            if (fiName.isPresent()) {
-                emitFunctionalInterfaceGetter(fiName.get(), javaName);
-            }
+            fiName.ifPresent(s -> emitFunctionalInterfaceGetter(s, javaName));
+        } else {
+            throw new IllegalArgumentException("Tree type not handled: " + varTree.type());
```

Comment:
Note that here I'm adding an explicit default branch for the if/else blocks above (which is never taken).